docs/middleware.md

```diff
@@ -0,0 +1,76 @@
+# Using Next 12 middleware on Netlify
+
+Next 12 introduces a new feature called [middleware](https://nextjs.org/docs/middleware), which are functions that are
```

Comment:
```suggestion
Next 12 introduces a new feature called [Middleware](https://nextjs.org/docs/middleware), which are functions that are
```

Reply:
@MelanieCrissey Would the section title need to change too?

> # Using Next 12 Middleware on Netlify

Sounds like we're wanting it capitalized when it refers to the feature itself